README.md

```diff
@@ -111,6 +111,15 @@ class MyController : public MjbotsControlLoop<LcmLog, LcmInput, MyRobot>
 Refer to `include/examples/simple_robot.h` for a sample robot class and 
 `examples/robot_example.cpp` for a usage example.
 
+## Behaviors
```

Comment:
Add some documentation on how to create the derived class.

Reply:
Added in f5235b4a9f2f70179410d9f36c16d79c5d13f04e